app/src/pages/Group/components/MembersTable/MembersTable.jsx

```diff
@@ -21,6 +21,7 @@ function MembersTable({ members, isLoading }) {
       },
       {
         key: 'displayName',
+        label: 'Display name',
```

Comment:
Can you just make this "Name"?
 
Oh and while you're at it, mind checking if i missed any other tables' "displayName" column? :D